client/src/assets/Navigation.tsx

```diff
@@ -0,0 +1,48 @@
+import { useState } from "react";
+
+export default function Navigation() {
+  const [active, setActive] = useState("Main"); // default active button
+
+  const buttons = ["Main", "Saved Posts", "Settings"];
```

Comment:
```suggestion
const BUTTONS = ["Main", "Saved Posts", "Settings"] as const;
const DEFAULT_ACTIVE = BUTTONS[0]
export default function Navigation() {
  const [active, setActive] = useState(DEFAULT_ACTIVE); // default active button
```

요로코롬 컴포넌트 바깥에 상수로 정의해서 사용할 수 있답니다!